cpp/server/ct-server.cc

```diff
@@ -159,7 +160,8 @@ static bool ValidatePort(const char* flagname, int port) {
 static const bool port_dummy =
     RegisterFlagValidator(&FLAGS_port, &ValidatePort);
 
-static bool ValidateRead(const char* flagname, const string& path) {
+static bool ValidateRead(const char* flagname __attribute__((unused)),
```

Comment:
Surprisingly, no... But then the docs for that attribute say that is means that "...this variable is meant to possibly be unused, GCC *[and clang, empirically]* will not produce a warning for this variable."

That said, I'm not over the moon about the extra grit in the lotion these attributes bring (I originally like the documentation aspect of it), so I think I might just remove the parameter names instead.